DOCS-3840 Search and Manage Synthetic Tests Edits + Unified Service Tagging for Synthetics Documentation (#14762)
DataDog/documentation: content/en/getting_started/tagging/unified_service_tagging.md

   **Note**: The PHP Tracer does not support configuration of unified service tagging for logs.

[1]: /tracing/other_telemetry/connect_logs_and_traces/
   {{% /tab %}}

   {{% tab "RUM & Session Replay" %}}

   If you're using [connected RUM and traces][1], specify the browser application in the `service` field, define the environment in the `env` field, and list the versions in the `version` field of your initialization file. 

   When you [create a RUM application][2], confirm the `env` and `service` names.

[1]: /real_user_monitoring/connect_rum_and_traces/
[2]: /real_user_monitoring/browser/#setup
   {{% /tab %}}

   {{% tab "Synthetics" %}}

   If you're using [connected Synthetic browser tests and traces][1], specify a URL to send headers to under the **APM Integration for Browser Tests** section of the [Integration Settings page][2]. 

   You can use `*` for wildcards, for example: `https://*.datadoghq.com`.

[1]: /synthetics/apm/
[2]: https://app.datadoghq.com/synthetics/settings/integrations
   {{% /tab %}}

   {{% tab "Custom Metrics" %}}

>    Tags are added in an append-only fashion for [custom statsd metrics][1]. For example, if you have two different values for `env`, the metrics are tagged with both environments. There is no order in which one tag overrides another of the same name.

Reviewer: ```suggestion
   Tags are added in an append-only fashion for [custom StatsD metrics][1]. For example, if you have two different values for `env`, the metrics are tagged with both environments. There is no order in which one tag overrides another of the same name.
```

   If your service has access to `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`, then the DogStatsD client automatically adds the corresponding tags to your custom metrics.

>    **Note**: The Datadog DogStatsD clients for .NET and PHP do not yet support this functionality.

Reviewer: ```suggestion
   **Note**: The Datadog DogStatsD clients for .NET and PHP do not support this functionality.
```

[1]: /metrics/
   {{% /tab %}}

   {{% tab "System Metrics" %}}

   You can add `env` and `service` tags to your infrastructure metrics.

>    The tagging configuration for service metrics lives closer to the Agent in non-containerized contexts.

Reviewer: what does it mean for a configuration to "live closer" to the Agent?

Author: I don't know actually 😅

   Given that this configuration does not change for each invocation of a service's process, adding `version`
   to the configuration is not recommended.

>    #### Single service per host

Reviewer: this whole section seems to be weirdly indented. not sure if it makes a difference but should probably be dedented for consistency

   Set the following configuration in the Agent's [main configuration file][1]:

   ```yaml
   env: <ENV>
   tags:
       - service:<SERVICE>
   ```

   This setup guarantees consistent tagging of `env` and `service` for all data emitted by the Agent.

   #### Multiple services per host

   Set the following configuration in the Agent's [main configuration file][1]:

   ```yaml
   env: <ENV>
   ```

   To get unique `service` tags on CPU, memory, and disk I/O metrics at the process level, configure a [process check][2] in the Agent's configuration folder (for example, in the `conf.d` folder under `process.d/conf.yaml`):

   ```yaml
   init_config:
   instances:
       - name: web-app
         search_string: ["/bin/web-app"]
         exact_match: false
         service: web-app
       - name: nginx
         search_string: ["nginx"]
         exact_match: false
         service: nginx-web-app
   ```

>    **Note**: If you already have a `service` tag set globally in your Agent's main configuration file, the process metrics are tagged with two services. Since this can cause confusion with interpreting the metrics, it is recommended to configure the `service` tag only in the configuration of the process check.

Reviewer: ```suggestion
   **Note**: If you already have a `service` tag set globally in your Agent's main configuration file, the process metrics are tagged with two services. Since this can cause confusion when you are interpreting the metrics, it is recommended that you configure the `service` tag only in the configuration of the process check.
```

[1]: /agent/guide/agent-configuration-files
[2]: /integrations/process
   {{% /tab %}}
   {{< /tabs >}}

### Serverless environment

> For more information about AWS Lambda function, see [how to connect your Lambda telemetry using tags][13].

Reviewer: ```suggestion
For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][13].
```
